marlowe-chain-sync/src/Language/Marlowe/Runtime/ChainSync/Api.hs

```diff
@@ -251,23 +263,30 @@ fromJSONEncodedTransactionMetadata = \case
     pure (label, value')
   _ -> Nothing
 
+
+toCardanoTxMetadata :: TransactionMetadata -> C.TxMetadata
+toCardanoTxMetadata (TransactionMetadata metadata) = C.TxMetadata $ toCardanoMetadata <$> metadata
+
+fromCardanoTxMetadata :: C.TxMetadata -> TransactionMetadata
+fromCardanoTxMetadata (C.TxMetadata metadata) = TransactionMetadata $ fromCardanoMetadata <$> metadata
+
 -- | An input of a transaction.
 data TransactionInput = TransactionInput
```

Comment:
This is interesting - could you please clarify why you drop this strictness annotation?

Reply:
I enabled `StrictData` in the module, which makes strict fields the default